### PR TITLE
Fix interactive shell called from installer script

### PR DESCRIPTION
**Pull request by rene**
## Description

One fix and one small improvement in the installer script:

#### Forces shell to run interactively when required
    
The installer script runs within a onboot container, with output/input not redirected to the current tty console. When a pause is required by the user (before or after the installation process), the shell called within the install script will not work. Forcing input/output to /dev/console fixes the problem.
    
#### Move "pause before install" to early stage
    
The boot option "pause before install" will pause the installation process only after wiping disks (when required), so move the pause to an earlier stage, before any destructive action to storage devices is performed.

### CODE REVIEW

File: pkg/installer/install [hunk 1]
```diff
@@ -64,8 +64,8 @@ logmsg() {
 }
 
 pause() {
-   echo "Pausing before $1. Entering shell. Type 'exit' to proceed to $1"
-   sh
+   echo "Pausing before $1. Entering shell. Type 'exit' to proceed to $1" > /dev/console 2>&1
+   sh < /dev/console > /dev/console 2>&1
```
Comment (OhmSpectator) [inline pkg/installer/install:68]: We are 100% sure that `/dev/console` is available and has the proper permissions, right?

Comment (rene) [reply pkg/installer/install:68]: I think so, /dev/console always points to the system's console, also, if you look a few lines above (https://github.com/lf-edge/eve/pull/4299/files/30b4e8972e3ee04d9114c6c9108d69b9ebd1fc21#diff-d2570dd09cc66cad9c33cbf655c47811e5959ad5d2df67fabfb2ca2d44b293f7R63) that's how the installer script is printing information to the terminal and the log file.....

Comment (OhmSpectator) [reply pkg/installer/install:68]: Ah, yeah, makes sense

**Review by OhmSpectator — APPROVED** [review]
I believe, you have already tested it?

**Review by deitch — APPROVED** [review]

**Pull request merged**